WordPressKit/BlogServiceRemoteREST+Jetpack.swift

```diff
@@ -0,0 +1,55 @@
+public enum JetpackInstallError: String, Error {
+    case invalidCredentials = "INVALID_CREDENTIALS"
+    case forbidden = "FORBIDDEN"
+    case installFailure = "INSTALL_FAILURE"
+    case installResponseError = "INSTALL_RESPONSE_ERROR"
+    case loginFailure = "LOGIN_FAILURE"
+    case siteIsJetpack = "SITE_IS_JETPACK"
+    case activationOnInstallFailure = "ACTIVATION_ON_INSTALL_FAILURE"
+    case activationResponseError = "ACTIVATION_RESPONSE_ERROR"
+    case activationFailure = "ACTIVATION_FAILURE"
+    case unknown
+
+    init?(error code: String?) {
+        guard let code = code else {
```

Comment:
If you wanted to get rid of the `guard` you could add a default value to `code` like so:
`init?(error code: String = JetpackInstallError.unknown.rawValue)`

Reply:
At the end I removed the guard without using the default value. Since the `init(rawValue:)` returns an optional I can use the nil-coalescing  to use `.unknown` as fallback.